### ISSUE TITLE
Screen Reader: Many Pages: The text field is focused and announced twice

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77329]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Using Android TalkBack, open the Expensify app
2. Swipe to the 'Phone or email' text field
3. Listen to the Talkback announcement
4. Swipe one more time
5. Listen to the TalkBack announcement

Other Occurrences:
1. Onboarding: all fields in the flow
2. On Inbox: the field 'Enter your boss's email'
3. On Status: the field 'Message'
4. On Status - Vacation Delegate
5. On Profile - Display Name
6. On Profile - Address
7. On Profile - Legal Name
8. On Security - Merge Accounts
9. On Security - Close Account
10. On Wallet - Add Bank Account
11. On Save the World - I know a teacher
12. On Filters
13. On Create Workspace

## Expected Result:
The field should be focused and announced only once, e.g.:
- 'Phone or email input field, edit box'

## Actual Result:
The text field is focused and announced twice: at first only as text, then with the edit field role

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Ensure the field is focused only once, group the label and text as a single focus point.
https://appt.org/en/docs/react-native/samples/accessibility-group

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/8a7c9704-20c1-4d8d-8e89-6295454b1d21)

https://github.com/user-attachments/assets/bc9dc681-b024-468f-aa04-db3c43356c97

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7005433
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
done
